usr/mgmt_ipc.c

```diff
@@ -411,8 +411,14 @@ mgmt_ipc_write_rsp(queue_task_t *qtask, int err)
 	}
 
 	qtask->rsp.err = err;
-	if (write(qtask->mgmt_ipc_fd, &qtask->rsp, sizeof(qtask->rsp)) < 0)
-		log_error("IPC qtask write failed: %s", strerror(errno));
+	if (write(qtask->mgmt_ipc_fd, &qtask->rsp, sizeof(qtask->rsp)) < 0) {
+		if (qtask->conn && qtask->conn->session)
+			log_error("connect%d:%d IPC qtask write failed: %s",
```

Comment:
Why not use conn_error here?  "connect" vs. "connection"

Reply:
done